QWeb/internal/util.py

```diff
@@ -310,3 +311,11 @@ def escape_xpath_quotes(text):
     if '"' in text:
         return f"'{text}'"
     return f'"{text}"'
+
+
+def anchor_to_index(anchor):
```

Comment:
typing should be added to function definition

Reply:
Oops, yep, will do.

Reply:
done with the latest commit